StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/TextField/TextFieldElement+Card.swift

```diff
@@ -361,4 +361,23 @@ extension TextFieldElement {
             return TextFieldElement.PANConfiguration(cardBrandDropDown: cardBrandDropDown).accessoryView(for: lastFourFormatted, theme: theme)
         }
     }
+    struct USBankNumberConfiguration: TextFieldElementConfiguration {
```

Comment:
Let's put this in it's own file, it doesn't belong in the `+Card` file.

Reply:
Created a new file, TextFieldElement+Card.swift specifically for USBankNumberConfiguration